src/GitHub.Exports.Reactive/ViewModels/ViewModelItemContainer.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace GitHub.ViewModels
+{
+    /// <summary>
+    /// Simple container for complex objects that we stuff
+    /// into dropdowns and the likes
+    /// </summary>
+    public abstract class ViewModelItemContainer
```

Comment:
Hmm, this seems to be a rather general class. and the name doesn't really describe it very well. How about something like `NamedItem`?